spec/bookmarks-view-spec.coffee

```diff
@@ -128,7 +204,7 @@ describe "Bookmarks package", ->
         editor.setCursorBufferPosition([2, 0])
         atom.commands.dispatch editorElement, 'bookmarks:toggle-bookmark'
 
-        editor.setCursorBufferPosition([10, 0])
+        editor.setSelectedBufferRanges([[[8, 4], [10, 2]]])
```

Comment:
Since line 10 is empty, could you change this range to end at [10, 0]? This would then be consistent with the tests on lines 217, 231, 237, and 242.